docs/protocol/src/concepts/validators.md

```diff
@@ -3,44 +3,52 @@
+
+Third, the validator could be manually disabled by the operator. The validator is then in the disabled state.  It does not participate in consensus, and the stake in its delegation pool does not earn rewards (the validator's rates are held constant).  The stake in its delegation pool will enter an unbonding period at the time the validator becomes disabled. The only valid state a disabled validator may enter into is "inactive", if the operator re-activates it by updating the validator definition.
+
+Fourth, the validator could be displaced from the validator set by another validator with more stake in its delegation pool. The validator is then in the inactive state.  It does not participate in consensus, and the stake in its delegation pool does not earn rewards (the validator's rates are held constant).  The stake in its delegation pool will enter an unbonding period at the time the validator becomes inactive.  Inactive validators have three possible state transitions:
 
 1. they can become active again, if new delegations boost its weight back into the top N;
-2. they can be slashed, if evidence of misbehavior arises during the unbonding period;
-3. they can become inactive, if neither (1) nor (2) occurs before the unbonding period passes.
+2. they can be tombstoned, if evidence of misbehavior arises during the unbonding period;
+3. they can be disabled, if the operator chooses.
 
-If (2) occurs, the same state transitions as in regular slashing occur: all pending undelegations are cancelled, etc.
-If (3) occurs, all pending undelegations are immediately removed from quarantine, short-circuiting the unbonding period that began when the undelegation was performed.  If (1) occurs, the validator stops unbonding, but this has no effect on pending undelegations, since they were quarantined with an unbonding period that started when the undelegation was performed (i.e., as if they were undelegations from an active validator).
+If (2) occurs, the same state transitions as in regular tombstoning occur: all pending undelegations are cancelled, etc.
+If (3) occurs, the unbonding period continues and the validator enters the disabled state.
+If (1) occurs, the validator stops unbonding, and all delegations are no longer subject to unbonding restrictions.
```

Comment:
should this be ", and all delegations become bonded again"? since stopping unbonding => bonded ?